objects.go

```diff
@@ -181,6 +252,30 @@ func (rule *Rule) writeOperation(verb verb, ctx *nftContext, writer io.Writer) {
 	fmt.Fprintf(writer, "\n")
 }
 
+// groups---------------------------1-2-------3---4--------5----6------7---------8----
+var ruleExp = fmt.Sprintf(` %s( index %s)?( handle %s)? ([^"]*)( comment %s)?$`,
+	noSpaceGroup, numberGroup, numberGroup, commentGroup)
+
+func parseRule(line, prefix string) (*Rule, error) {
+	ruleRegexp := regexp.MustCompile(prefix + ruleExp)
```

Comment:
So I guess you did the `prefix` thing so you don't have to validate the family and table name here, but it seems kind of weird. It seems like it would be clearer to have the `add rule %s %s ` as part of `ruleExp`, etc.

You could either
1. have `parseRule`, etc, assume that the caller already checked the family and table name, so you can just ignore them here
2. pass an `nftContext` to `parseRule`, etc, and then have a helper function to validate that the regexp-parsed ip and family match the `nftContext`.